installation/scripts/testing-common.sh

```diff
@@ -217,6 +217,18 @@ spec:
   repositories:
   - url: "https://github.com/kyma-project/addons/releases/download/0.7.0/index-testing.yaml"
 EOF
+    local retry=0
+    while [[ ${retry} -lt 15 ]]; do
+        msg=$(kubectl get clusteraddonsconfiguration ${TESTING_ADDONS_CFG_NAME} -o=jsonpath='{.status.phase}')
+        if [[ ${msg} = "Ready" ]]; then
+            break
+        else
+            echo "waiting for ready testing addons..."
+            sleep 2
+        fi
+        retry=$[retry + 1]
+    done
+
```

Comment:
and what if status will not become Ready???

Reply:
added error handling